src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,29 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.Logger;
```

Comment:
The import for `Logger` should be from `org.apache.logging.log4j.Logger` instead of `org.apache.logging.log4j.core.Logger`. The `core` package is typically used for internal purposes and not for general logging usage.